boards/px4/fmu-v5x/nuttx-config/nsh/defconfig

```diff
@@ -250,7 +250,7 @@ CONFIG_UART5_IFLOWCONTROL=y
 CONFIG_UART5_OFLOWCONTROL=y
 CONFIG_UART5_RXBUFSIZE=600
 CONFIG_UART5_RXDMA=y
-CONFIG_UART5_TXBUFSIZE=1500
+CONFIG_UART5_TXBUFSIZE=3000
```

Comment:
We should port this to V5 (and all other relevant boards) as well?

Reply:
@mhkabir give it a quick try? If there's value in increasing it we could do it uniformly on all the TELEM ports on newer F7/H7 boards.

Reply:
I increased it as I tested with very high rate logging (~180KB/s). Given we have enough RAM we can increase it on all boards, but it will be very unusual that anyone will need it.